docs/misc/faq.md

```diff
@@ -101,7 +101,7 @@ with several powerful functionalities:
  where thousands of people have uploaded useful containers: anything from Redis,
```

Comment:
Sorry, couldn't comment on the line itself: the one to https://hub.docker.com. that will redirect to sign up if you're not logged in